.github/workflows/deprecation-check.yml

```diff
@@ -0,0 +1,42 @@
+name: Deprecation check
+
+on:
+  pull_request:
+    paths:
+      - packages/yoast-components/**
+      - packages/babel-preset/**
+      - packages/e2e-tests/**
+      - packages/helpers/**
+      - packages/style-guide/**
+
+# Cancels all previous workflow runs for the same branch that have not yet completed.
+concurrency:
+  # The concurrency group contains the workflow name and the branch name.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  notify-of-deprecations:
+    name: 'notify of changes in deprecated packages'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Place a comment on the PR
+        uses: thollander/actions-comment-pull-request@1d3973dc4b8e1399c0620d3f2b1aa5e795465308 # v2.4.3
```

Comment:
```suggestion
        uses: thollander/actions-comment-pull-request@v2.4.3
```

Let's use the published version 😄

Reply:
Actually, it was a conscious decision to lock on the SHA 😄 

From https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions:
> Pinning an action to a full length commit SHA is currently the only way to use an action as an immutable release. Pinning to a particular SHA helps mitigate the risk of a bad actor adding a backdoor to the action's repository, as they would need to generate a SHA-1 collision for a valid Git object payload. When selecting a SHA, you should verify it is from the action's repository and not a repository fork.

Reply:
Wow, TIL 😮 Thanks! 🙏